test/e2e_node/density_test.go

```diff
@@ -0,0 +1,490 @@
+/*
+Copyright 2016 The Kubernetes Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
```

Comment:
The boilerplate is wrong. If you compare this with other files in the repository, you'll notice that an empty line missing before an after the URL. They probably got lost during copy/paste. Need to fix that to pass the tests.